renderer/ffmpeg-convert.js

```diff
@@ -0,0 +1,49 @@
+const ffmpegPath = require("ffmpeg-static-electron").path;
+const { spawn } = require("child_process");
+var path = require("path");
+
+const AUDIO_VIDEO_FORMATS = [
```

Comment:
Why do we have audio and video formats together? We would want to split these up. Are there any ambiguous audio/video formats?

Reply:
Audio and video formats are together because in certain cases converting between them is quite handy (think mp4 -> mp3). FFmpeg has no problem with it. Also, webm is one such format that is used for both Audio and Video.

Reply:
Got it. Makes sense 👍